src/core/text/TextStyle.js

```diff
@@ -232,6 +235,19 @@ export default class TextStyle
         }
     }
 
+    get fillGradientStops()
+    {
+        return this._fillGradientStops;
+    }
+    set fillGradientStops(fillGradientStops)
+    {
+        if (this._fillGradientStops !== fillGradientStops)
```

Comment:
Will do; also think I've seen an issue with the stop gradients settings not working nice enough over multiple lines, so I'll take a look when I get time :)